src/libinjector/linux/linux_injector.c

```diff
@@ -290,6 +316,16 @@ bool init_injector(injector_t injector)
             return load_shellcode_from_file(injector, injector->host_file);
             break;
         }
+        case INJECT_METHOD_EXECPROC:
+        {
+            if (!injector->host_file)
+            {
+                fprintf(stderr, "Inject file is required\n");
```

Comment:
Why not PRINT_DEBUG?

Reply:
Coz it should be visible to the user without debug build also